docs/applications/configuration-management/getting-started-with-ansible/index.md

```diff
@@ -0,0 +1,240 @@
+
+    If your Ansible managed infrastructure will change over time, it is recommended to use the [dynamic inventory plugin for Linode](https://docs.ansible.com/ansible/latest/plugins/inventory/linode.html). You can read the [How to use the Linode Ansible Module to Deploy Linodes](/docs/applications/configuration-management/deploy-linodes-using-ansible/) to learn how to use this plugin.
+
+* **Modules**: Modules add extra functionality to Ansible. You can call Ansible modules directly from the command line to execute on your managed nodes or use them in your Playbooks. See [Ansible's module index](https://docs.ansible.com/ansible/latest/modules/modules_by_category.html) for a list of available modules by category.
+
+* **Tasks**: The simplest unit of execution in Ansible is a task. Tasks utilize Ansible modules to manage your host's services, packages, files and to perform various system configurations. Tasks can be executed from the command line or within Playbooks.
+
+* **Playbooks**: Playbooks are YAML files containing a list of tasks in the desired order of execution. You can run Playbooks on your managed nodes and reuse and share them. [Variables](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html) and [Jinja templating](https://docs.ansible.com/ansible/latest/user_guide/playbooks_templating.html) provide a powerful way to execute complex tasks on your managed hosts.
+{{</ disclosure-note >}}
+
+## Scope of this Guide
+This guide introduces the basics of installing Ansible and preparing your environment to use [Playbooks](https://docs.ansible.com/ansible/latest/user_guide/playbooks.html). You will complete the following steps in this guide:
+
+* Install and configure Ansible on your computer or a Linode to serve as the control node that will manage your infrastructure nodes.
+* Create two Linodes to manage with Ansible and establish a basic connection between the control node and your managed nodes. The managed nodes will be referred to as `node-1`, and `node-2` throughout the guide.
+
+    {{< note >}}
+The examples in this guide provide a manual method to establish a basic connection between your control node and managed nodes as a way to introduce the basics of Ansible. If you would like to learn how to use Ansible's [Linode module](https://docs.ansible.com/ansible/latest/modules/linode_v4_module.html) to automate deploying and managing Linodes, see the [How to use the Linode Ansible Module to Deploy Linodes](/docs/applications/configuration-management/deploy-linodes-using-ansible/). The guide assumes familiarity with Ansible modules, Playbooks, and dynamic inventories.
+    {{</ note >}}
+
+## Before You Begin
+
+{{< caution >}}
+This guide's example instructions will create up to three billable Linodes on your account. If you do not want to keep using the example Linodes that you create, be sure to [delete them](#delete-a-cluster) when you have finished the guide.
```

Comment:
I added "up to" three billable here because we say that they could 1) run the following with just one managed node and 2) they could make their home computer the control node.